src/plugins/type/type.c

```diff
@@ -203,10 +203,15 @@ static kdb_long_long_t readBooleanRestore (KeySet * config)
 
 	const char * restoreString = keyString (restore);
 
+	if (strcmp (restoreString, "none") == 0)
+	{
+		return -2;
```

Comment:
It would be nice if readBooleanRestore has some docu, e.g. from the meaning of the return values.